index.html

```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
 			<a class="see" href="#"> See Project</a>
 		</div>
 	</div>
+
+	<!-- destop cards -->
+	<div class="card hide">
+		<div class="row first-row">
+			<div class="cards">
+				<img src="assets/images/first-col.png" alt="rec">
+				<a class="see" href="#"> See Project</a>
+			</div>
+			<div class="cards card-2">
+				<h1 class="work-sample"> Data Dashboard Health care </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+			<div class="cards card-3">
+				<br><br>
+				<h1 class="work-sample"> Website Protfolio </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+		</div>
+		<div class="row first-row">
+			<div class="cards card-1">
+				<h1 class="work-sample"> Profesional Art Printing Data </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+			<div class="cards card-2">
+				<h1 class="work-sample"> Data Dashboard Health care </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+			<div class="cards card-3">
+				<br><br>
+				<h1 class="work-sample"> Website Protfolio </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+		</div>
+	</div>
+	</div>
 </section>
 <br> <br> <br>
 <section class="head">
 	<!-- Heading Section  -->
 	<div class="flex-about">
-	  <div class="about-me">
+	<div class="row">
+	  <div class="about-me col-7">
```

Comment:
- [ ] It has some white space left in the section please fix it.

![image](https://user-images.githubusercontent.com/38635158/173108200-97b5cd27-24b3-4ad5-a55d-efea254f5b4f.png)

- [ ] Please align the center of the get my resume button's text. Also, apply hover, disable and active style on it.
- [ ] This section layout is broken in the mobile view. Please fix it.
![image](https://user-images.githubusercontent.com/38635158/173108753-8eadebe9-3c6d-4235-882e-5276dc18a50d.png)
___
Original
![image](https://user-images.githubusercontent.com/38635158/173109934-9a34afdd-a34d-4dba-ab93-858103811f47.png)

Reply:
Thank you for the corrections, I applied that. Please check it by Inspect -> mobile devices -> Responsive